t/script/release/skip_broken_files.t

```diff
@@ -0,0 +1,42 @@
+use strict;
+use warnings;
+
+use Archive::Tar;
+use HTTP::Request;
+use LWP::UserAgent;
+use MetaCPAN::Script::Release;
+use Test::More;
+
+use Test::RequiresInternet ( 'metacpan.org' => 443 );
+
+my $ua = LWP::UserAgent->new(
+    parse_head => 0,
+    env_proxy  => 1,
+    agent      => 'metacpan',
+    timeout    => 30,
+);
+
+my $URL
```

Comment:
`my $url`  Reserve upper case variable names for situations which require constants.